pom.xml

```diff
@@ -360,7 +355,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>3.3.0</version>
+        <!-- Version specified in parent POM -->
```

Comment:
Unrelated minor cleanup: I noticed this version was redundant while adding the new entry for `maven-surefire-plugin`. Note that the version for `maven-shade-plugin` _does_ need to be specified, since it is not included in the parent POM.

Reply:
https://github.com/jenkinsci/pom/issues/371